conf/modules.config

```diff
@@ -691,13 +781,86 @@ process{
 
 // PAIR_VARIANT_CALLING
 
+    //ASCAT
+    withName: 'ASCAT' {
+        ext.args = {
+            [
+            "gender": meta.sex,
+            //"genomeVersion": "hg19"
+            //"purity": (optional),
+            //"ploidy": params.ploidy,
+            //"gc_files": (optional),
+            //"minCounts": (optional),
+            //"chrom_names": (optional),
+            //"min_base_qual": (optional),
+            //"min_map_qual": (optional),
+            //"ref_fasta": (optional),
+            //"skip_allele_counting_tumour": (optional),
+            //"skip_allele_counting_normal": (optional)
+            ]
+        }
+    }
+
+    //CONTROLFREEC
+    withName: 'MPILEUP_NORMAL' {
+        ext.prefix = { "${meta.id}.normal" }
+    }
+
+    withName: 'MPILEUP_TUMOR' {
+        ext.prefix = { "${meta.id}.tumor" }
+    }
+
+    withName: 'CAT_MPILEUP_NORMAL' {
+        ext.prefix = { "${meta.id}.normal.pileup" }
```

Comment:
shouldn't the `.pileup` be part of the file extension in the module?

Reply:
only happens if the default is used but not a custom prefix :/ I tried to use the logic for automatically taking the last index. Couldn't get it to work immediatly and went with typing it instead. I'll see if I can make that work

Reply:
From the module:
```
    prefix   = task.ext.prefix ?: "${meta.id}${file_list[0].substring(file_list[0].lastIndexOf('.'))}"
    out_zip  = prefix.endsWith('.gz')
    in_zip   = file_list[0].endsWith('.gz')
    command1 = (in_zip && !out_zip) ? 'zcat' : 'cat'
    command2 = (!in_zip && out_zip) ? "| pigz -c -p $task.cpus $args2" : ''
    """
    $command1 \\
        $args \\
        ${file_list.join(' ')} \\
        $command2 \\
        > ${prefix}
       ```

Reply:
Oh, I see, no worries then...